js/ui/messageList.js

```diff
@@ -115,8 +117,8 @@ var URLHighlighter = new Lang.Class({
         }));
     },
 
-    setMarkup: function(text, allowMarkup) {
-        text = text ? _fixMarkup(text, allowMarkup) : '';
+    setMarkup: function(text, allowMarkup, onlySimpleMarkup) {
```

Comment:
I have verified that all users of `setMarkup` are here in `js/ui/messageList.js` before doing the signature change, but I agree. Even when JS allows to call methods with less parameters, and leaves the remaining paramters set to `null` which is falsey.

Checking for `this._bodySimpleMarkup` inside is not an option, because that property is from Message and `setMarkup` is in class `URLHighlighter`. So I will use a default value.